airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/avro/JsonToAvroSchemaConverter.java

```diff
@@ -73,7 +77,7 @@ static List<JsonSchemaType> getTypes(final String fieldName, final JsonNode fiel
     }
```

Comment:
Should `airbyteTypeProperty` be passed to the `JsonSchemaType#fromJsonSchemaType` call in the `if` block above as well? I think that's needed to process elements in the array.

Reply:
Seems we cannot specify airbyteTypeProperty to array, only for single JSON field, so no value for passing.